docs/content/en/docs/pipeline-stages/builders/ko.md

```diff
@@ -216,27 +217,115 @@ Useful tips for existing `ko` users:
   file. If you already configure your base images using this file, you do not
   need to repeat this configuration in `skaffold.yaml`.
 
-- When you use the Skaffold ko builder, Skaffold takes care of replacing the
-  image placeholder name in your Kubernetes manifest files using its
-  [render]({{< relref "/docs/pipeline-stages" >}}) functionality.
-
-- The ko builder supports image name placeholders that consist of the
-  `ko://` prefix, followed by the Go import path to the main package.
-  This means that Skaffold should work with existing Kubernetes manifest
-  files that use this image name placeholder format.
-
-  If you previously rendered Kubernetes manifest using `ko`, e.g.:
+- Generate a starter `skaffold.yaml` file by searching your existing manifests
+  for image references starting with `ko://` by using this snippet:
 
   ```shell
-  ko resolve --filename k8s/*.yaml > out.yaml
+  cat << EOF > skaffold.yaml
+  apiVersion: skaffold/v2beta26
+  kind: Config
+  build:
+    artifacts:
+  EOF
+  grep -rho "ko://$(go list -m)[^\"]*" ./config/ | sort | uniq | xargs -Iimg echo -e "  - image: img\n    ko: {}" >> skaffold.yaml
+  cat << EOF >> skaffold.yaml
+    local:
+      concurrency: 0
+    tagPolicy:
+      sha256: {}
```

Comment:
We can remove this tagPolicy

Reply:
The reason I added it is because this is a section specifically to help existing `ko` users. `ko` users tend to always use digests rather than tags, and this setting means that when Skaffold renders manifests, it'll render the same way as `ko`.

Reply:
The downside to the `sha256:` tagger is that it always tags as `latest`.  [`inputDigest`](https://skaffold.dev/docs/pipeline-stages/taggers/#inputdigest-uses-a-digest-of-the-artifact-source-to-tag-images) might be better.

(We have `sha256` throughout our examples.  I have a dangling patch somewhere to rename it to `latest` — maybe that would be good to bring in for Skaffold 2.0.)